apps/site/pages/en/learn/modules/package-configuration.md

```diff
@@ -0,0 +1,563 @@
+  - If your source-code does _not_ contain any top-level `await`, your ESM likely can be `required()`, meaning it can be consumed by a project using CJS. See [`require()` an ES Module](https://nodejs.org/api/modules.html#loading-ecmascript-modules-using-require).
+
+| You as a package author write     | Consumers of your package write their code in                    | Your options                                                                                                                                                 |
+| :-------------------------------- | :--------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| CJS source code using `require()` | CJS: consumers `require()` your package                          | [CJS source and distribution](#cjs-source-and-distribution)                                                                                                  |
+| CJS source code using `require()` | ESM: consumers `import` your package                             | [CJS source and only ESM distribution](#cjs-source-and-only-esm-distribution)                                                                                |
+| CJS source code using `require()` | CJS & ESM: consumers either `require()` or `import` your package | [CJS source and both CJS & ESM distribution](#cjs-source-and-both-cjs-amp-esm-distribution)                                                                  |
+| ESM source code using `import`    | CJS: consumers `require()` your package                          | [ESM source with only CJS distribution](#esm-source-with-only-cjs-distribution)<br />[ESM source and distribution](#esm-source-and-distribution)             |
+| ESM source code using `import`    | ESM: consumers `import` your package                             | [ESM source and distribution](#esm-source-and-distribution)                                                                                                  |
+| ESM: source code uses `import`    | CJS & ESM: consumers either `require()` or `import` your package | [ESM source and both CJS & ESM distribution](#esm-source-and-both-cjs-amp-esm-distribution)<br />[ESM source and distribution](#esm-source-and-distribution) |
+
+### CJS source and distribution
+
+This the "Rum & Coke" of packages: pretty difficult to mess up. You _technially_ may not need _any_ package configuration beyond [`"name"`](https://nodejs.org/api/packages.html#name). But the less arcane, the better: Essentially just declare the package’s exports via the `"exports"` field/field-set.
+
+**Working example**: [cjs-with-cjs-distro](https://github.com/JakobJingleheimer/nodejs-module-config-examples/tree/main/packages/cjs/cjs-distro)
+
+```json displayName="Minimal package.json"
+{
+  "name": "cjs-source-and-distribution"
+  // "main": "./index.js"
```

Comment:
It's commented to show it's not needed because node will pick it up automatically (assuming the most basic setup). I did it because you wanted a minimal package.json with the least config possible (this is how you get that).

Reply:
I think in practice, removing the main field already means extra configuration for people. From what I can tell, at least >50% of the high impact packages use the main field instead of the exports field, which is likely due to the fact that `npm init` just adds the main field by default, so removing that requires extra actions. I would say the "minimal configuration" should be using the main field instead.